### ISSUE TITLE
java.lang.UnsatisfiedLinkError crash

### ISSUE THREAD

**Issue by olexale** [talsec/Free-RASP-Flutter#73]
**Describe the bug**
This might be a duplicate for https://github.com/talsec/Free-RASP-Flutter/issues/59. The crash happened in freeRASP 4.0.0, but I haven't found a mention in the changelog for 5.0.x that it is fixed, so I'm reporting the issue.

For some users, the app crashed with the following stacktrace:
```
java.lang.UnsatisfiedLinkError: No implementation found for java.lang.String[] com.aheaditec.talsec_security.security.Natives.b() (tried Java_com_aheaditec_talsec_1security_security_Natives_b and Java_com_aheaditec_talsec_1security_security_Natives_b__)
	at com.aheaditec.talsec_security.security.Natives.b(Native Method)
	at com.aheaditec.talsec_security.security.Natives.i(SourceFile:1)
	at com.aheaditec.talsec.security.e1.b(SourceFile:3)
	at com.aheaditec.talsec.security.b1.t(SourceFile:1)
	at com.aheaditec.talsec.security.b1.H(SourceFile:5)
	at com.aheaditec.talsec.security.b1.p(SourceFile:1)
	at com.aheaditec.talsec.security.a1.run(R8$$SyntheticClass:5)
	at com.aheaditec.talsec.security.e.b(SourceFile:9)
	at com.aheaditec.talsec.security.b1.y(SourceFile:6)
	at com.aheaditec.talsec.security.b1.a(SourceFile:2)
	at com.aheaditec.talsec.security.l2.e(SourceFile:31)
	at com.aheaditec.talsec.security.l2.a(SourceFile:1)
	at com.aheaditec.talsec.security.k2.run(R8$$SyntheticClass:5)
	at java.lang.Thread.run(Thread.java:919)
```

**To Reproduce**
Unfortunately, I can not reproduce the crash locally.

**Expected behavior**
The app not crashes.

**Please complete the following information:**
 - Device: Xiaomi Nexus 6P
 - OS version: Android 10
 - Version of freeRASP: 4.0.0

**Comment by yardexx** [comment]
Hi!

We are aware of this issue and we working on it. Fix should be available on next release.

**Comment by msikyna** [comment]
Hello @olexale ,
we have inspected a similar issue and provided a partial solution and suggested a few more solutions, which needs to be tested. You can check it out in this issue: https://github.com/talsec/Free-RASP-Android/issues/18

Kind regards,
Talsec team

**Comment by yardexx** [comment]
Closed as resolved.

Issue was resolved since freeRASP 6.0.0

If issue persists or reoccurs, please reopen this issue.
